clients/drcachesim/analyzer.cpp

```diff
@@ -122,6 +122,15 @@ analyzer_t::record_is_timestamp(const memref_t &record)
         record.marker.marker_type == TRACE_MARKER_TYPE_TIMESTAMP;
 }
 
+template <>
+void
+analyzer_t::set_wait_record(memref_t &record)
+{
+    record.marker.type = TRACE_TYPE_MARKER;
+    record.marker.marker_type = TRACE_MARKER_TYPE_WAIT;
+    record.marker.marker_value = 0;
```

Comment:
Make a symbolic constant or add a comment about the zero.

Reply:
Changed it to memset the other fields.